### PR TITLE
improve Japanese translation for 5.9

### PR DESCRIPTION
**Pull request by h1data**
# Description

Improving Japanese localization for 5.9
Resolves: #19438

## Type of change

- [x] Translations

# How Has This Been Tested?

- [x] Compiled *.mo file by gettext and checked the appearance of the built application.

**Test Configuration**:
* Operating System: Windows 11

# Checklist:

- [x] I have read the [Contribution guide](https://github.com/Ultimaker/Cura/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by h1data** [comment]
@wawanbreton Here is the Japanese translation updated for 5.9.

Our changes in #19473 were reverted in 546df30,
so I updated the changes again with additional corrections since #19473.
Lokalize folks might have only applied template *.pot files except revised *.po files.
Could you inform them about the translation changes?

**Comment by wawanbreton** [comment]
Thanks ! I'll take a look at the PR quickly.

> Could you inform them about the translation changes?

Not really, I am not in direct contact with the translators. However I am the one running the procedure to send the labels to be translated. This procedure is still being debugged, so maybe there is still something wrong with it. Or maybe I just got confused because your PR arrived at the same time as the translations and I did something wrong.

File: resources/i18n/ja_JP/cura.po [hunk 1]
```diff
@@ -44,113 +44,114 @@ msgstr "%1m"
 
 msgctxt "@action:inmenu menubar:printer"
 msgid "&Add Printer..."
-msgstr "＆プリンターを追加する..."
+msgstr "プリンターを追加(&A)..."
```
Comment (wawanbreton) [inline resources/i18n/ja_JP/cura.po:47]: I love the idea of restoring the `&A`, but just for my curiosity, is it useable at all with a Japanese keyboard ?

Comment (h1data) [reply resources/i18n/ja_JP/cura.po:47]: Yes, it's the traditional style for shortcuts in Japanese applications.
image

Of course, I have confirmed it works in 5.9.1 like `Alt+a`.

Comment (wawanbreton) [reply resources/i18n/ja_JP/cura.po:47]: Ok nice, thanks for the explanation :slightly_smiling_face:

**Pull request merged**